tests/test_router_bgp.rb

```diff
@@ -256,24 +257,18 @@ def test_asnum_dot
   end
 
   def test_nsr
+    skip_if_UnsupportedCmdRef('bgp', 'nsr')
```

Comment:
Looks like the nsr stuff was added by the XR team when they were still doing puppet.  I see that NSR is supported for  OSPFv2 on Nexus but there is no mention of BGP.  Did you find documentation that states it's not supported for BGP or is it the absence of the CLI?

Reply:
Neither. Our yaml excludes it so I added this check to obey the yaml. (sushirol exluded it from nexus when he added the attr)